parsely-amp-tracking.php

```diff
@@ -0,0 +1,25 @@
+add_action( 'pre_amp_render_post', 'parsely_amp_add_actions' );
+function parsely_add_amp_actions() {
```

Comment:
This will become `public function parsely_add_amp_actions()` once we've got it within the `Parsely` class